LiskHQ/lisk-desktop
# Incorrect unlock times

### What was the problem?

This PR resolves #5049

### How was it solved?

Use the `expectedUnlockTime` field from BE to determine when a stake can be unlocked

### How to test?

1. Stake 100 with a validator
2. Edit that stake to 80
3. Go to http://0.0.0.0:8080/#/validators/profile/stakes?modal=lockedBalance
4. You should see a text with "20 LSK" - "will be available to unlock in a month"
